### DISCUSSION TITLE
在Table的EditTemplate中使用Row,ColSpan不起作用

### DISCUSSION THREAD

**Discussion by LostRoy** [dotnetcore/BootstrapBlazor#3242] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

在Table的EditTemplate中,使用了Row控制 ItemsPerRow="ItemsPerRow.Four",内部再嵌套Row使 ColSpan="4" ,但是没有起作用
![1712563440753](https://github.com/dotnetcore/BootstrapBlazor/assets/34463769/0ac30cfa-3dd6-4379-b98a-96cc132053cf)

### Steps To Reproduce

以下是直接使用的 TableDemo
```
<Table TItem="Foo" AutoGenerateColumns="true" class="table-demo"
    IsPagination="true" PageItemsSource="PageItemsSource"
    IsStriped="true" IsBordered="true" IsMultipleSelect="true" EditDialogRowType="RowType.Inline"
    ShowSkeleton="true" ShowToolbar="true" ShowSearch="true" ShowExtendButtons="true"
    OnQueryAsync="OnQueryAsync" OnSaveAsync="OnSaveAsync" OnDeleteAsync="OnDeleteAsync">
    <TableColumns>
        <TableColumn @bind-Field="@context.Hobby" Items="GetHobbys(context)" />
    </TableColumns>

    <EditTemplate Context="value">
        <Row ItemsPerRow="ItemsPerRow.Four" RowType="RowType.Inline">
            <BootstrapInput @bind-Value="@value.Name" />
            <BootstrapInput @bind-Value="@value.Name" />
            <BootstrapInput @bind-Value="@value.Name" />
            <BootstrapInput @bind-Value="@value.Name" />
            <Row ColSpan="4" RowType="RowType.Inline">
                <BootstrapInput @bind-Value="@value.Name" />
            </Row>
            <BootstrapInput @bind-Value="@value.Name" />
            <BootstrapInput @bind-Value="@value.Name" />
            <BootstrapInput @bind-Value="@value.Name" />
        </Row>
    </EditTemplate>

</Table>
```

### .NET Version

8.0

### Anything else?

BootstrapBlazor 8.*

**Comment by bb-auto** [comment]
@LostRoy Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@LostRoy 这种复杂的逻辑放到 `EditTemplate` 中本身就一些缺陷，建议单独封装成一个组件

```
<EditTemplate>
    <CustomComponent />
</EditTemplate>
```

`Row` 组件有一些小问题，并不完善，因为复杂的布局用这个并不能实现，建议使用 `Bootstrap` 的布局

https://getbootstrap.com/docs/5.3/layout/grid/

请帮忙点一下右上角 `Star` 并点击👇🏻 `make as answer`
